price-feeder.example_forex.toml

```diff
@@ -0,0 +1,163 @@
+gas_adjustment = 1
+
+[server]
+listen_addr = "0.0.0.0:7171"
+read_timeout = "20s"
+verbose_cors = true
+write_timeout = "20s"
+
+[[deviation_thresholds]]
+base = "USDT"
+threshold = "1.5"
+
+[[deviation_thresholds]]
+base = "ATOM"
```

Comment:
I'm not really a big fan of having two files here tbh - if we have a section in the price feeder config docs on how to add a forex asset, I think that will suffice 

Also, might want to split this into another PR